e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubLegacyPublisherDeployments.cs

```diff
@@ -0,0 +1,87 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.Deploy {
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using TestExtensions;
+
+    public sealed class IoTHubLegacyPublisherDeployments : DeploymentConfiguration {
+
+        /// <summary>
+        /// MessagingMode that will be used for configuration of OPC Publisher.
+        /// </summary>
+        public readonly MessagingMode MessagingMode;
+
+        /// <summary>
+        /// Create deployment
+        /// </summary>
+        /// <param name="context"></param>
+        public IoTHubLegacyPublisherDeployments(IIoTPlatformTestContext context) : base(context) {
+        }
+
+        /// <inheritdoc />
+        protected override int Priority => 1;
```

Comment:
This sets the same priority for `IoTHubLegacyPublisherDeployments` as `IoTHubPublisherDeployment`. Shall they be the same ? Or should we set it to `2` here ? On a side note, `IoTHubEdgeBaseDeployment` has a priority of `0`.

Reply:
Set priority = 2 just to avoid any issue